.github/workflows/e2e.yml

```diff
@@ -92,7 +92,7 @@ jobs:
     - name: Run tests
       env:
         BASE_URL: https://${{ secrets.NGROK_ADDRESS }}
-        RACK_ENV: production
+        RACK_ENV: e2e_test
```

Comment:
We moved it from `e2e_test` to `production` earlier because we missed some issues that caused downtime. We're trying to keep the E2E pipeline similar to production.

Reply:
If you need to check if it's running E2E tests and accept the consequences of different behaviors for PG cases, we could introduce a new config variable like `Config.is_e2e` and check its value in PG workflows.